ambry-file-transfer/src/main/java/com/github/ambry/FileCopyManager.java

```diff
@@ -41,8 +41,7 @@ class PartitionStateChangeListenerImpl implements PartitionStateChangeListener {
 
     @Override
     public void onPartitionBecomeBootstrapFromOffline(String partitionName) {
-      // StateBuilding at the end of FCM's Offline->Bootstrap transition
-      storeManager.buildStateForFileCopy(partitionName);
+      // StateBuilding (storeManager.buildStateForFileCopy()) will be triggered at the end of FCM's async handler.
```

Comment:
Please change the method name to something generic in the actual interface. Like buildBlobStoreState(). thos should be a generic method and scope should not be limited to FileCopy in future use cases.

Reply:
Or let's call it buildReplicaStateInMemory(), to make it even more generic

Reply:
@aga9900 I think we should keep the name as is since it is building state specific to FileCopy (only for sealed segments, ignoring active segments). It wouldn't be reused elsewhere since there is no generic usecase.

Reply:
sounds good.